src/new/wallet.js

```diff
@@ -566,7 +570,12 @@ class HathorWallet extends EventEmitter {
    **/
   handleWebsocketMsg(wsData) {
     if (wsData.type === 'wallet:address_history') {
-      this.onNewTx(wsData);
+      if ([HathorWallet.PROCESSING, HathorWallet.SYNCING].includes(this.state)) {
+        // Cannot receive a ws message, so we enqueue for later
```

Comment:
I guess this comment is imprecise. Maybe "cannot process new transactions from ws when the wallet is not ready. Queuing it to be processed later."?

Reply:
Done

Reply:
Thanks!